### PR TITLE
Encode multi-frame Tiff format

### PR DESCRIPTION
**Pull request by IldarKhayrutdinov**
### Prerequisites

- [x] I have written a descriptive pull-request title
- [x] I have verified that there are no overlapping [pull-requests](https://github.com/SixLabors/ImageSharp/pulls) open
- [x] I have verified that I am following the existing coding patterns and practice as demonstrated in the repository. These follow strict Stylecop rules :cop:.
- [x] I have provided test coverage for my change (where applicable)

### Description

Saving multi-frame Tiff images.

### CODE REVIEW

**Comment by JimBobSquarePants** [comment]
Awesome! Looking forward to having a read through!

**Comment by IldarKhayrutdinov** [comment]
it seems that now can reviewing, don't hesitate to comments 😄

**Comment by JimBobSquarePants** [comment]
Excellent! Thanks @IldarKhayrutdinov I'll have a look later tonight.

File: src/ImageSharp/Common/Helpers/UnitConverter.cs [hunk 1]
```diff
@@ -100,12 +100,11 @@ public static PixelResolutionUnit ExifProfileToResolutionUnit(ExifProfile profil
         /// <summary>
         /// Sets the exif profile resolution values.
         /// </summary>
-        /// <param name="exifProfile">The exif profile.</param>
         /// <param name="unit">The resolution unit.</param>
         /// <param name="horizontal">The horizontal resolution value.</param>
         /// <param name="vertical">The vertical resolution value.</param>
         [MethodImpl(InliningOptions.ShortMethod)]
-        public static void SetResolutionValues(ExifProfile exifProfile, PixelResolutionUnit unit, double horizontal, double vertical)
+        public static (ushort, double?, double?) AdjustToExif(PixelResolutionUnit unit, double horizontal, double vertical)
```
Comment (IldarKhayrutdinov) [inline src/ImageSharp/Common/Helpers/UnitConverter.cs:107]: I don't really like the signature of this method (return type), I'll be glad to suggestions

**Review by IldarKhayrutdinov — COMMENTED** [review]
At the moment the frame settings will be the same for each frames (Tiff can save each frame with its own compression, photometric ...)
https://github.com/SixLabors/ImageSharp/blob/cad7340d5b47d0ecc897583b2d90525e49ac9fd7/src/ImageSharp/Formats/Tiff/TiffEncoder.cs#L21-L37

File: src/ImageSharp/Common/Helpers/UnitConverter.cs (hunk 1 above)
Comment (JimBobSquarePants) [reply src/ImageSharp/Common/Helpers/UnitConverter.cs:107]: No that seems off both in naming of the method and the return type. I'd return an explicitly declared readonly struct.

Comment (IldarKhayrutdinov) [reply src/ImageSharp/Common/Helpers/UnitConverter.cs]: Done! Please review

**Review by JimBobSquarePants — APPROVED** [review]
Thanks @IldarKhayrutdinov this looks good to me. 

I'll let @brianpopow sense check though before merging just to be sure since you guys are the codec experts.

**Review by brianpopow — APPROVED** [review]
looks good, thank you @IldarKhayrutdinov!

**Pull request merged**